internals/cli/cli.go

```diff
@@ -214,92 +208,57 @@ func Parser(cli *client.Client) *flags.Parser {
 		}
 		cmd.PassAfterNonOption = c.PassAfterNonOption
 
-		opts := cmd.Options()
-		if c.OptionsHelp != nil && len(opts) != len(c.OptionsHelp) {
-			logger.Panicf("wrong number of option descriptions for %s: expected %d, got %d", c.Name, len(opts), len(c.OptionsHelp))
-		}
-		for _, opt := range opts {
-			name := opt.LongName
-			if name == "" {
-				name = string(opt.ShortName)
-			}
-			desc, ok := c.OptionsHelp[name]
-			if !(c.OptionsHelp == nil || ok) {
-				logger.Panicf("%s missing description for %s", c.Name, name)
-			}
-			lintDesc(c.Name, name, desc, opt.Description)
-			if desc != "" {
-				opt.Description = desc
+		optionsHelp := map[string]string{}
+		positionalArgsHelp := map[string]string{}
+
+		for specifier, help := range c.ArgsHelp {
+			if strings.HasPrefix(specifier, "--") {
+				optionsHelp[specifier] = help
+			} else if utf8.RuneCountInString(specifier) == 2 && strings.HasPrefix(specifier, "-") {
+				optionsHelp[specifier] = help
```

Comment:
How do you handle options that have a `--longName` and shorthand `-l` with the same effect?